notebook/static/notebook/js/cell.js

```diff
@@ -114,9 +114,15 @@ define([
             readOnly: false,
             theme: "default",
             extraKeys: {
-                "Cmd-Right":"goLineRight",
-                "End":"goLineRight",
-                "Cmd-Left":"goLineLeft"
+                "Cmd-Right": "goLineRight",
+                "End": "goLineRight",
+                "Cmd-Left": "goLineLeft",
+                "Tab": "indentMore",
+                "Shift-Tab" : "indentLess",
+                "Cmd-Alt-[" : "indentAuto",
+                "Ctrl-Alt-[" : "indentAuto",
+                "Cmd-/" : "toggleComment",
+                "Ctrl-/" : "toggleComment",
```

Comment:
I was going to suggest adding a block comment shortcut, but I checked CodeMirror.commands, and it's not available :sob: